app/src/components/formdemo/FormDemo.tsx

```diff
@@ -750,6 +752,148 @@ export default function FormDemo() {
 		),
 	});
 
+	const tableProps = {
+		className: 'table',
+		headerRow: {
+			cells: [
+				{
+					renderedContent: <p>Observation Type</p>,
```

Comment:
I made a change so it's no longer necessary to do this. You can only provide `content` if you don't want it rendered differently than the simple value. But `renderedContent` will be preferred if it's set.

Reply:
So content isn't a required prop?

Reply:
The TS definition is correct;`content` is required, `renderedContent` is not. https://github.com/purecloudlabs/genesys-react-components/pull/18/commits/84b1e2418e46fc23cd58c441de33051bf7c4b78c

Reply:
Do you prefer it removed in the demo?

Reply:
Yes, demonstrate it the way it's meant to be used. Only provide `content` if it's a simple value. Use `content` and `renderedContent` to demonstrate the case where the content used for sorting/filtering is different from what's displayed.